### DISCUSSION TITLE
Corner circle

### DISCUSSION THREAD

**Discussion by kobzar** [elkowar/eww#863] [Q&A]
HI 2 all.
I am need a sample how to add two top corners widget. Left for the workspace (active in center) and
right for tasks (nmapplet in center)
Something like this
![corners](https://github.com/elkowar/eww/assets/1912842/cf4cb8f9-70e3-4fe4-9e06-d544f4d62470)

**Comment by elkowar** [comment]
Make two windows, use css to set border-radius to make them round, position your window content _probably_ via absolute positioning by using the transform widget -- should be fine

**Reply by mantacid** [reply]
Agreed. If your window width isn’t going to change (not that there’s any way it could), just open a square window at the corner of the screen (`:x “0%” :y “0%”` for top left, `:x “100%” :y “0%” `for top right), set border radius to 50% on the widget, and place buttons with a transform.
